lib/src/drm/common/utils/stringToArray.ts

```diff
@@ -0,0 +1,3 @@
+export function stringToArray(str: string): Uint16Array {
+	return new Uint16Array([...str].map(char => char.charCodeAt(0)));
+}
```

Comment:
This function should be moved to the root utility folder, and renamed `stringToUint16`